specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/examples/getPendingSubscriptionOperations.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
can we rename this file to "getSubscriptionOperations.json"? I think we'll also want to return (recently) completed operations for a better developer experience.

Reply:
done